README.rst

```diff
@@ -157,7 +154,7 @@ Of course!
     "Second User, Third User",Third User,2013-10-04T15:05Z,2
     ...
 
-What about that YAML thing the kids these days are talking about?
+What about that YAML?
```

Comment:
heh, this can probably just say "What about YAML?". Also, PyPI's restructured-text parser is very pedantic, so the line of `~` underneath must be the same length as what it's underlining.